### PR TITLE
Create ordinary-objects_korman-daniel.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/papers/ordinary-objects_korman-daniel.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Ordinary Objects"
+authors:
+  - "Daniel Z. Korman"
+external_url: "https://plato.stanford.edu/entries/ordinary-objects/"
+drive_links:
+  - "https://plato.stanford.edu/entries/ordinary-objects/"
```
Comment (buddhist-uni) [inline _content/papers/ordinary-objects_korman-daniel.md:7]: That's no drive link! :death_star:

**Review by buddhist-uni — COMMENTED** [review]
Welcome back! Hope you had a good rest of your holiday 😊

Glad to see you figured out how to open the PR from the Desktop software!

File: _content/papers/ordinary-objects_korman-daniel.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+course: emptiness
+tags:
+  - philosophy
+year: 2011
+publisher: "Stanford Encyclopedia of Philosophy"
```
[the remark this answers is not in the document]
Comment (buddhist-uni) [reply _content/papers/ordinary-objects_korman-daniel.md:12]: https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_content/articles/chan_hershock-peter.md?plain=1

File: _content/papers/ordinary-objects_korman-daniel.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Ordinary Objects"
+authors:
+  - "Daniel Z. Korman"
+external_url: "https://plato.stanford.edu/entries/ordinary-objects/"
+drive_links:
+  - "https://plato.stanford.edu/entries/ordinary-objects/"
+course: emptiness
+tags:
```
[the remark this answers is not in the document]
Comment (JJM317) [reply _content/papers/ordinary-objects_korman-daniel.md:9]: I'd go with perception over ideology. For me, ideology is more a set of beliefs, etc. Perception, besides the physical function, can be taken in the way this paper uses it.

File: _content/papers/ordinary-objects_korman-daniel.md (hunk 1 above)
Comment (JJM317) [reply _content/papers/ordinary-objects_korman-daniel.md:7]: ya, sorry :(

**Review by buddhist-uni — APPROVED** [review]

**Pull request merged**